src/anm/utils.js

```diff
@@ -199,6 +197,13 @@ function fit_rects(pw, ph, aw, ah) {
     } else return [ 1, [ 0, 0, aw, ah ] ];
 }
 
+function removeElement(array, element) {
```

Comment:
Hmm, it wasn't the array operation at all. It was a method to [delete properties in the object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete).